florist/api/launch.py

```diff
@@ -0,0 +1,155 @@
+import logging
```

Comment:
What do you think about putting this file in a subdirectory and leave `api` for the files that have APIs exposed? Call it `launchers` or `process` or something like that, maybe?